app/controllers/pxe_controller/pxe_image_types.rb

```diff
@@ -15,11 +15,6 @@ def pxe_image_type_edit
     if params[:button] == "cancel"
```

Comment:
This whole `cancel` branch can be dropped, not just the message building. We're handling the cancel on the frontend completely, together with the redirect.